### PR TITLE
NEXUS-8361 - Create ExtJS facet views which add fields based on what the...

### PR DESCRIPTION
**Pull request by kellyrob99**
- update httpclient config to include authentication and optional connection details

screen shot 2015-04-20 at 2 49 11 pm
screen shot 2015-04-20 at 2 49 28 pm

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/HttpFacet.js [hunk 1]
```diff
@@ -23,33 +23,113 @@ Ext.define('NX.coreui.view.repository.facet.HttpFacet', {
...
       },
       {
-        xtype: 'numberfield',
-        name: 'httpclient.connection.timeout',
-        fieldLabel: NX.I18n.get('ADMIN_REPOSITORIES_SETTINGS_CONNECTION_TIMEOUT'),
-        helpText: NX.I18n.get('ADMIN_REPOSITORIES_SETTINGS_CONNECTION_TIMEOUT_HELP'),
-        minValue: 0,
-        value: 0
+        xtype: 'nx-optionalfieldset',
+        title: NX.I18n.get('LEGACY_ADMIN_REPOSITORIES_SETTINGS_HTTP'),
```
Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/HttpFacet.js]: LEGACY?

**Comment by adreghiciu** [comment]
+1

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/HttpFacet.js (hunk 1 above)
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/HttpFacet.js]: Whoops, will update

**Comment by jdillon** [comment]
+1

**Pull request merged**